lib/jazzy/source_declaration/access_control_level.rb

```
@@ -5,26 +5,52 @@ class AccessControlLevel
 
       attr_reader :level
 
-      def initialize(declaration_string)
-        if declaration_string =~ /private\ /
+      ACCESSIBILITY_PRIVATE  = 'source.lang.swift.accessibility.private'
+      ACCESSIBILITY_INTERNAL = 'source.lang.swift.accessibility.internal'
+      ACCESSIBILITY_PUBLIC   = 'source.lang.swift.accessibility.public'
+
+      def initialize(accessibility)
+        if accessibility == ACCESSIBILITY_PRIVATE
           @level = :private
-        elsif declaration_string =~ /public\ /
-          @level = :public
-        else
+        elsif accessibility == ACCESSIBILITY_INTERNAL
           @level = :internal
+        elsif accessibility == ACCESSIBILITY_PUBLIC
+          @level = :public
+        end
```

should probably throw an exception if its something unexpected?